playwright-e2e/pages/osteo/release-page.ts

```diff
@@ -0,0 +1,150 @@
+
+  await page.getByRole('button', { name: '+ Add another physician' }).click();
+  await page.locator('#mat-input-22').click();
+  await page.locator('#mat-input-22').fill('Dr. Feelgood');
+  await page.locator('#mat-input-19').click();
+  await page.locator('#mat-input-19').fill('Heavy Metal Hospital');
+  await page.locator('#mat-input-20').click();
+  await page.locator('#mat-input-20').fill('Big Hair');
+  await page.locator('#mat-input-20').press('Tab');
+  await page.locator('#mat-input-21').fill('MA');
+  await page.locator('#mat-input-23').click();
+  await page.locator('#mat-input-23').fill('USA');
+  await page.locator('section').filter({ hasText: 'Thank you for your consent to participate in this research study. To complete th' }).click();
+ 
+  await page.getByText('I have already read and signed the informed consent document for this study, whi').click();
+  await page.getByLabel('Full Name').click();
+
+  await page.getByText('Thank you for your consent to participate in this research study.').click();
+  await page.getByLabel('Physician Name').click();
+  await page.getByLabel('Physician Name').fill('Dr. Teeth');
+  await page.getByLabel('Physician Name').press('Tab');
+  await page.getByRole('combobox', { name: 'Institution (if any)' }).fill('institute');
+  await page.getByRole('combobox', { name: 'Institution (if any)' }).press('ArrowDown');
+  await page.getByRole('combobox', { name: 'Institution (if any)' }).press('ArrowDown');
+  await page.getByRole('combobox', { name: 'Institution (if any)' }).press('ArrowDown');
+  await page.getByRole('combobox', { name: 'Institution (if any)' }).press('Enter');
+  await page.getByRole('button', { name: '+ Add another physician' }).click();
+  await page.locator('#mat-input-22').click();
+  await page.locator('#mat-input-22').fill('Dr. Feelgood');
+  await page.locator('#mat-input-19').click();
+  await page.locator('#mat-input-19').fill('Heavy Metal Hospital');
+  await page.locator('#mat-input-20').click();
+  await page.locator('#mat-input-20').fill('Big Hair');
+  await page.locator('#mat-input-20').press('Tab');
+  await page.locator('#mat-input-21').fill('MA');
+  await page.locator('#mat-input-23').click();
+  await page.locator('#mat-input-23').fill('USA');
+  await page.locator('section').filter({ hasText: 'Thank you for your consent to participate in this research study. To complete th' }).click();
+  await page.getByText('I have already read and signed the informed consent document for this study, whi').click();
+  await page.getByLabel('Full Name').click();
+  await page.getByRole('combobox', { name: 'Full Name' }).fill('Andrew Zimmer');
+  
+  */
+
+  async waitForReady(): Promise<void> {
+    await expect(this.pageTitle).toBeVisible({ visible: true });
+    await expect(this.pageTitle).toHaveText('Medical Release');
+    await waitForNoSpinner(this.page);
+  }
+
+  async bloodAndTissue(agreeToBlood: boolean, agreeToTissue: boolean) : Promise<void> {
```

Comment:
since this is the release page, I think the above comment stuff is probably for here and the below is for the consent (in github, mostly sure it was the file I saw before this one)

Reply:
Good catch, I'll mop this up.